docs/development/sub.md

```diff
@@ -0,0 +1,29 @@
+# SubjuGator
+This page provides general info about developing for [RoboSub](https://robonation.org/programs/robosub/), an anual competition for autonomous submarines.
+MIL has participated in RoboSub for 20+ years. Most code for this is hosted under SubjuGator/.
+
+## Running SubjuGator in Simulation
+
+### Getting Setup
```

Comment:
Since this is copy pasted from VRX (bad practice), let's move this section to Development Guide and reference it here (e.g "First, enter the development environment (See Development Guide)"

Reply:
Please see most recent push, I have updated the Docs to remedy this.